client/src/components/MagicalFactor/keywords.js

```diff
@@ -0,0 +1,22 @@
+const keywordsData = {
+  solo: { text: 'Solo traveller friendly', src: 'solo' },
+  wildlife: { text: 'Great for wildlife', src: 'wild' },
+  family: { text: 'Family owned', src: 'family' },
```

Comment:
I might be missing sth, but does this actually work? Seems to me that `src` will resolve to a string and not the path to the image. Anyways, I would import the images from the assets folder here instead of that index.js file and used them directly in the `src` field.